packages/integration-sdk-core/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@jupiterone/integration-sdk-core",
+  "version": "0.1.0",
```

Comment:
At some point along the path, these versions will all match?

Reply:
Yeah, after all of the monorepo changes have been merged, we'll version all of the packages and then publish.